interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -5,84 +5,61 @@ Vault Nomination
-   
-   * Note: in the usual case, this should be handled by having the different collateral thresholds (secure, premium redeem, liquidation). But in extreme cases (very high exchange rate volatility), it might cause concern.
-
 
 
 Vault Nomination Protocol
 -------------------------
 
-#. Vaults can choose to opt in and out of the Nomination protocol. If they opt-in they take on the additional role of an Operator.
-#. Nominators select an Operator to which they can delegate DOT balance as collateral. As a reward, they will earn a fraction of the interbtc and DOT fees generated by this collateral. The other fraction of these fees is received by the Operator.
-#. Vault replacement is disallowed for Operators with nominated collateral. Otherwise, Security Assumptions 1 and 2 would be violated.
-#. The nominated DOT:
-
-   #. Is locked on the parachain
-
-   #. Cannot be withdrawn by the operating Vault
-
-   #. Is capped at a fraction of the Vault’s deposited collateral (Max Nomination Ratio). This prevents the Operator from withdrawing its entire collateral and only exposing Nominators to economic risk, or stealing without liquidation consequences. This means that an Operator can only withdraw collateral as long as the fraction of nominated collateral does not exceed the threshold cap. Capping Nominator collateral also prevents Operators being “outnumbered” by Nominators and their relative fee earnings being marginalized.
-
-#. Liquidation slashing is handled as follows.
-
-   #. In case the collateral managed by the Operator falls below the liquidation threshold, the Operator and Nominators are slashed proportionally to their collateral.
-   
-   #. In case the Operator steals Bitcoin deposited at its address, its collateral is used to cover as much of the slashed amount as possible. If the Operator’s collateral was not enough to cover the entire amount, the Nominators are slashed proportionally for the remaining amount.
-
-#. Collateral withdrawals are first requested and then executed. A withdrawal request:
-
-   #. Decreases the issuable interbtc capacity.
-   
-   #. May be cancelled if not executed. The amount in the cancelled withdrawal request becomes backing collateral again.
-   
-   #. Is subject to a window of delay (unbonding period) that allows Nominators and the Operator to react.
+#. Vaults must choose to opt in to the Nomination protocol.
+#. Nominators select a Vault to which they can delegate collateral. They will earn a fraction of any rewards generated by the Vault.
+#. Vault replacement is disallowed with nominated collateral. Otherwise, Security Assumptions 1 and 2 would be violated.
+#. The nominated collateral:
 
-      #. Operator Unbonding Period. This window is longer, because Nominators are assumed to be mostly offline.
+   #. Cannot be withdrawn by the Vault,
+   #. Is locked on the parachain,
+   #. Is capped at a fraction of the Vault’s deposited collateral (:ref:`maxNominationRatio`). 
```

Comment:
```suggestion
   #. Is capped at a fraction of the Vault’s deposited collateral (:ref:`maxNominationRatio`). This prevents the Vault from withdrawing its entire collateral and only exposing Nominators to economic risk, or stealing without liquidation consequences. This means that a Vault can only withdraw collateral as long as the fraction of nominated collateral does not exceed the threshold cap. Capping Nominator collateral also prevents Vaults from being “outnumbered” by Nominators and their relative fee earnings being marginalized.
```

Reply:
I've moved this to the "Max Nomination Ratio" section below.